spark/interpreter/pom.xml

```diff
@@ -517,60 +517,6 @@
           </execution>
         </executions>
       </plugin>
-
-
-      <plugin>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>copy-dependencies</id>
-            <phase>none</phase>
-            <configuration>
-              <skip>true</skip>
-            </configuration>
-          </execution>
-
-          <execution>
-            <id>copy-interpreter-dependencies</id>
-            <phase>none</phase>
-            <configuration>
-              <skip>true</skip>
-            </configuration>
-          </execution>
-          <execution>
-            <id>copy-artifact</id>
-            <phase>none</phase>
-            <configuration>
-              <skip>true</skip>
-            </configuration>
-          </execution>
-
-
-          <execution>
-            <id>copy-spark-interpreter</id>
-            <phase>package</phase>
```

Comment:
This copy operation is performed with the maven-shade-plugin.
At the moment we get the following output during the build.
```
2021-09-27T04:39:20.4923321Z [INFO] --- maven-dependency-plugin:3.1.2:copy (copy-spark-interpreter) @ spark-interpreter ---
2021-09-27T04:39:20.4925219Z [INFO] Configured Artifact: org.apache.zeppelin:spark-interpreter:0.11.0-SNAPSHOT:jar
2021-09-27T04:39:20.5007691Z [INFO] org.apache.zeppelin:spark-interpreter:0.11.0-SNAPSHOT:jar already exists in /home/runner/work/zeppelin/zeppelin/spark/interpreter/target/../../../interpreter/spark
```